hoomd/md/NeighborList.cc

```diff
@@ -1103,20 +1101,16 @@ void NeighborList::addOneThreeExclusionsFromTopology()
 
         if (nBondsA >= MAXNBONDS)
             {
-            m_exec_conf->msg->error()
-                << "nlist: Too many bonds to process exclusions for particle with tag: " << tagA
-                << endl;
-            m_exec_conf->msg->error() << "Maximum allowed is currently: " << MAXNBONDS - 1 << endl;
```

Comment:
Do we not want to inform users of the maximum bonds they can have?

Reply:
Done.